### PR TITLE
[T2A2][T09-A2]Tham Shi Yuan

### PR DESCRIPTION
**Pull request by thamsy**
Ready for Review.

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -141,9 +142,10 @@
      * used by the internal String[] storage format.
      * For example, a person's name is stored as the 0th element in the array.
      */
-    private static final int PERSON_DATA_INDEX_NAME = 0;
-    private static final int PERSON_DATA_INDEX_PHONE = 1;
-    private static final int PERSON_DATA_INDEX_EMAIL = 2;
+    private static final String PERSON_PROPERTY_NAME = "name";
```
Comment (nirandiw) [inline src/seedu/addressbook/AddressBook.java]: Isn't a enum better?

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1146,12 +1152,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
```
Comment (nirandiw) [inline src/seedu/addressbook/AddressBook.java]: Header comments needs to be updated to reflect the modified behaviour. 

When you modify the behavior, remember to update documentation and tests as well.

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -1146,12 +1152,12 @@ private static String getUsageInfoForExitCommand() {
...
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String fullString, String prefix) {
+        return fullString.replaceFirst(prefix, "");
```
Comment (nirandiw) [inline src/seedu/addressbook/AddressBook.java]: Shouldn't there be a validation to check if fullString.indexOf(prefix) == 0 before you actually do the replace?? If not the first occurrence of the prefix is what is getting replaced, right?

File: README.md [hunk 4]
```diff
@@ -83,12 +83,12 @@ Format: `list`
 > Finds persons that match given keywords
```
Comment (nirandiw) [inline README.md:83]: commit messages should be in imperative tone

**Review by nirandiw — COMMENTED** [review]
@thamsy Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/AddressBook.java (hunk 3 above)
Comment (thamsy) [reply src/seedu/addressbook/AddressBook.java]: I see yes indeed, I have changed that locally

File: README.md (hunk 4 above)
Comment (thamsy) [reply README.md:83]: I see, thank you, will do so for future commits

**Pull request closed without merging**